README.md

```diff
@@ -1,11 +1,5 @@
 # Model Predictive Path Integral Controller
 
-## Differential drive  
-![](.resources/demo-diff.gif)
```

Comment:
I actually rather like these gifs. I think they just need to be compressed so they take less room on memory

Reply:
Visually showing something working is a really valuable asset -- its often overlooked but is what makes users more compelled to try something new